property_caching/__init__.py

```diff
@@ -38,12 +45,14 @@ def _get_cache(obj, cache_attr_name=_OBJ_CACHE_ATTR_NAME):
     return getattr(obj, cache_attr_name, {})
 
 
-def _get_property_value(fn, obj, cache_attr_name):
+def _get_property_value(fn, obj, cache_attr_name, only_cache_true_vals=False):
+    result = None
```

Comment:
...and I have to reference it for the case where a truthy cached property is not in the cache yet and is not a truthy value.